src/index.ts

```diff
@@ -97,35 +108,78 @@ export default function rollupPluginSbom(userOptions?: RollupPluginSbomOptions):
         async moduleParsed(moduleInfo) {
             // filter out modules that exists in node_modules and
             // also are not Rollup virtual modules (starting with \0)
-            const nodeModuleImportedIds = moduleInfo.importedIds.filter(
-                (entry) => entry.includes("node_modules") && !entry.startsWith("\0"),
-            );
+            const resolutions = [
+                ...moduleInfo.importedIdResolutions,
+                ...moduleInfo.dynamicallyImportedIdResolutions,
```

Comment:
I should also add a test-case to verify if dyanmic imports are listed correctly.